dotnet/src/Plugins/Plugins.Web/Bing/BingConnector.cs

```diff
@@ -23,14 +22,19 @@ public sealed class BingConnector : IWebSearchEngineConnector
     private readonly ILogger _logger;
     private readonly HttpClient _httpClient;
     private readonly string? _apiKey;
+    private readonly string? _uri;
+    private const string DefaultUri = "https://api.bing.microsoft.com/v7.0/search?q";
 
     /// <summary>
     /// Initializes a new instance of the <see cref="BingConnector"/> class.
     /// </summary>
     /// <param name="apiKey">The API key to authenticate the connector.</param>
+    /// <param name="uri"></param>
```

Comment:
nit: Add some details.

Reply:
Resolved in https://github.com/microsoft/semantic-kernel/pull/3357/commits/9be54798d946929a1e09881aa6c31315cffb5114